src/resources/postcss/help-hub.pcss

```
@@ -63,6 +63,10 @@ ul.tec-help-list__list-expanded {
 
 .tec-settings__sidebar-icon-wrap-content {
 	margin-left: var(--tec-spacer-2);
+
+	p {
+		margin: 0 0 1em 0;
```

```suggestion
		margin: 0 0 1em;
```
Don't need the trailing `0`.